Source/ASTextNode2.mm

```diff
@@ -234,9 +234,12 @@ - (CGSize)calculateSizeThatFits:(CGSize)constrainedSize
 
   _textContainer.size = constrainedSize;
   [self _ensureTruncationText];
-  
+
+  // If constrained width is max/inf, the text node is calculating its intrinsic size.
+  const BOOL calculatingIntrinsicSize = (_textContainer.size.width >= ASTextContainerMaxSize.width);
```

Comment:
Does Texture ever use the `isVerticalForm` property? We should probably just delete it, but if not maybe check for it here.